docs/docs/welcome.mdx

```diff
@@ -31,30 +31,30 @@ import Demo from '../examples/Demo.js';
 [![Star on GitHub][github-star-badge]][github-star]
 [![Tweet][twitter-badge]][twitter]
 
-Reactjs-popup is a simple react popup component that help you create simple and complex Modals, tooltips and Menus for your next React App.
+`reactjs-popup` is a simple React popup component that help you create simple and complex Modals, Tooltips and Menus for your next React App.
```

Comment:
I capitalized "Tooltips" firstly to make it match "Modals" and "Menus", secondly because I do like the way it stands out. They should not be capitalized if we're following correct grammar but the capitalization makes them stand out and I do personally like the effect.

Let me know if I should remove the capitalization instead.